README.md

```diff
@@ -44,27 +41,37 @@ To get the big picture, read [PostgreSQL Hybrid Search Using pgvector](https://w
 
 Before you start working with pgai, you need:
 
-* An [OpenAI API Key](https://platform.openai.com/api-keys).
-* A postgres client like [psql v16](https://www.timescale.com/blog/how-to-install-psql-on-mac-ubuntu-debian-windows/) or [PopSQL](https://docs.timescale.com/use-timescale/latest/popsql/)
+* An [OpenAI API Key](https://platform.openai.com/api-keys) - everyone
+* A postgres client like [psql v16](https://docs.timescale.com/use-timescale/latest/integrations/query-admin/psql/) or [PopSQL](https://docs.timescale.com/use-timescale/latest/popsql/)
+* [Python3](https://www.python.org/downloads/) - if you prefer code to pure sql 
+
+## Create a pgai environment
 
-## How to get pgai
+The fastest ways to run PostgreSQL with TimescaleDB and pgai are:
 
-### Use a pre-built Docker image
+* [Enable pgai in a pre-built Docker container](#enable-pgai-in-a-pre-built-docker-container)
+* [Enable pgai in a Timescale Cloud service](#enable-pgai-in-a-timescale-cloud-service)
 
-1. Follow [these instructions](https://docs.timescale.com/self-hosted/latest/install/installation-docker/) 
-   to use pgai in docker with a pre-built image.
+### Enable pgai in a pre-built Docker container
 
-2. Create the pgai extension:
+1.  [Run the TimescaleDB Docker image](https://docs.timescale.com/self-hosted/latest/install/installation-docker/).
+
+1. Connect to your database:
+   ```bash
+   psql -d "postgres://<username>:<password>@<host>:<port>/<database-name>"
+   ```
+   
+1. Create the pgai extension:
 
     ```sql
     CREATE EXTENSION IF NOT EXISTS ai CASCADE;
     ```
 
    The `CASCADE` automatically installs the plpython3u and pgvector dependencies.
```

Comment:
These steps are common to all installation methods, so I would move that down to the common usage section below.

Reply:
To create the pgai extension you have to login to the database before you run CREATE command. If I leave out the connection string, our beloved user will have a "HUH?" moment. I'll leave as is.